include/fastdds/rtps/security/logging/Logging.h

```diff
@@ -137,6 +144,10 @@ class Logging
   bool options_set_ = false;
 
   LogOptions log_options_;
+  GUID_t guid_;
+  std::string guid_str_;
+  uint32_t domain_id_ = std::numeric_limits<uint32_t>::max();
```

Comment:
Are you certain this is an invalid domain? Might consider making this signed just so it can hold an uninitialized value.

Reply:
No 'max' is not an invalid value per se, I define that and also check for it in the function body. The codebase uses `unsigned int` for that variable thus I'd rather prefer to stick to it.

Reply:
Alright. Hopefully that doesn't bite us later :wink: .